util/src/main/scala/scala/scalanative/util/Scope.scala

```diff
@@ -39,21 +41,34 @@ object Scope {
       throw new UnsupportedOperationException("Can't close forever Scope.")
   }
 
+  /** Unsafe manually managed scope.*/
+  def unsafe: Scope = new Impl {}
```

Comment:
This should have `()` because it creates a different one each time:
```suggestion
  /** Unsafe manually managed scope.*/
  def unsafe(): Scope = new Impl {}
```